src/angular-business/table/table/table-data-source.ts

```diff
@@ -372,6 +391,10 @@ export class SbbTableDataSource<
    * @docs-private
    */
   connect() {
+    if (!this._renderChangesSubscription) {
```

Comment:
Don't we have to support disconnect?

  /**
   * Used by the SbbTable. Called when it connects to the data source.
   * @docs-private
   */
  connect() {
    if (!this._renderChangesSubscription) {
      this._updateChangeSubscription();
    }

    return this._renderData;
  }

Reply:
You're absolutey right